src/python/gudhi/wasserstein/wasserstein.py

```diff
@@ -79,6 +81,9 @@ def _perstot(X, order, internal_p, enable_autodiff):
         transparent to automatic differentiation.
     :type enable_autodiff: bool
     :returns: float, the total persistence of the diagram (that is, its distance to the empty diagram).
+
+    .. note::
+        Can be +infty if the diagram has an essential part (points with infinite coordinates).
```

Comment:
Is the abbreviation "infty" standard in python? "inf" seems more common, if you don't want to spell the whole word.

Reply:
Changed `infty` for `inf` in the doc. Should I also changed functions' names?

Reply:
I don't care much what the private functions are called.